recipes/fmt/all/conanfile.py

```diff
@@ -5,8 +5,7 @@
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 from conan.tools.files import get, apply_conandata_patches, copy, rmdir
 
-
-required_conan_version = ">=1.50.0"
+required_conan_version = ">=1.51.3"
```

Comment:
1.50.0 is enough I believe

Reply:
One of the TODO linked to an issue that was fixed in 1.51.3 from the comments

Reply:
Ok, seems quite excessive to bump required_conan_version for a bug occurring at consume time in only one legacy generator (cmake_paths).

Reply:
Yes, usually I wouldn't but I  am trying to get a few "perfect" examples from popular projects to show case.

So thank you for the help!